zapcli/zap_helper.py

```diff
@@ -394,5 +394,5 @@ def html_report(self, file_path):
     @staticmethod
     def _write_report(report, file_path):
         """Write report to the given file path."""
-        with open(file_path, 'w') as f:
+        with open(file_path, mode='wb') as f:
             f.write(report)
```

Comment:
I am checking it right now - but how did you notice this is not working? Look like the build is passing, and I checked - it is also running on Python 3.5...